subgraphs/burrow/src/handlers/actions.ts

```diff
@@ -236,8 +274,22 @@ export function handleWithdraw(event: EventData): void {
     .times(token.lastPriceUSD!);
 
   // usage metrics
-  if (account._last_active_timestamp.lt(usageDailySnapshot.timestamp)) {
+  const hourlyActiveAccount = ActiveAccount.load(
+    "HOURLY-"
+      .concat(account.id)
+      .concat("-")
+      .concat(NANOS_TO_HOUR(receipt.block.header.timestampNanosec).toString())
+  );
+  const dailyActiveAccount = ActiveAccount.load(
+    "DAILY-"
+      .concat(account.id)
+      .concat("-")
+      .concat(NANOS_TO_DAY(receipt.block.header.timestampNanosec).toString())
+  );
+  if (dailyActiveAccount == null) {
```

Comment:
these null comparisons are redundant. You can just say `if (!dailyActiveAccount)`

Reply:
It is redundant, and you use the `!` syntax in the rest of your code (https://github.com/prasad-kumkar/messari-subgraphs/blob/burrow/subgraphs/burrow/src/helpers/market.ts#L24)

I am okay with whatever you use, but whatever you choose should stay constant throughout your code